### DISCUSSION TITLE
Is there an option to turn off the minimap?

### DISCUSSION THREAD

**Discussion by octokatherine** [suren-atoyan/monaco-react#225] [Q&A] [answered]
Is there a way to not show the minimap on the side of the editor?

**Comment by hekar** [comment, marked as answer]
It's a property of `options`

```
  <Editor
     ...
     options={{
       minimap: {
         enabled: false,
       },
     }}
   />
```

More options:

- https://github.com/suren-atoyan/monaco-react/blob/master/src/types.d.ts#L90
- https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.istandaloneeditorconstructionoptions.html#minimap

**Reply by heldersepu** [reply]
those 2 links are 404
